generator/src/test/java/com/scottlogic/deg/generator/cucumber/LessThan.feature

```diff
@@ -2,12 +2,332 @@ Feature: User can specify that a numeric value is lower than, but not equal to,
 
 Background:
      Given the generation strategy is full
-      And there is a field foo
-      And foo is of type "numeric"
-      And foo is anything but null
+       And there is a field foo
+       And foo is of type "numeric"
+       And foo is anything but null
 
+# alone
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: User requires integer data
+     Given foo is less than 100
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo         |
+       | -2147482624 |
+       | -2147482623 |
+       | -2147482622 |
+       | -2147482621 |
+       | -2147482620 |
+
+@ignore #589: produces big-integer numbers rather than integer values, not sure why
+Scenario: User requires decimal data
+     Given foo is less than 100.1
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
+       | foo           |
+       | -2147482624.0 |
+       | -2147482623.9 |
+       | -2147482623.8 |
+       | -2147482623.7 |
+       | -2147482623.6 |
```

Comment:
unsure of these expected results, I would have expected:
100.0
99.9
99.8
99.7
99.6

Reply:
Interesting, these values will be produced, but with the current implementation will do produced last.

This sounds like a question for the PO on how it should work.

Reply:
Ref #594